package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "a5e",
   "version": "0.8.12",
+  "type": "module",
```

Comment:
I don't know if this will break anything, but it's the way I allowed es6 syntax in `lint-staged.config.js`